### ISSUE TITLE
Request to add Crown Copyright logo to the footer

### ISSUE THREAD

**Issue by johnbenjaminphe** [surevine/govuk-react-jsx#61]
Could you please include the Crown Copyright logo to the footer?
The project [coronavirus-dashboard](https://github.com/PublicHealthEngland/coronavirus-dashboard) has a requirement for this.

You can find it in the bottom right corner of this website [Public Health England - Fingertips](https://fingertips.phe.org.uk/)

**Comment by andymantell** [comment]
Will have a look this evening...

**Comment by andymantell** [comment]
Hi John,

Thanks for raising this. I've submitted a pull request against the coronavirus-dashboard here in order to fix it:

https://github.com/PublicHealthEngland/coronavirus-dashboard/pull/4

The reason it was missing is that webpack doesn't know where to find the assets - you have to tell put the right location into the $govuk-asset-path sass variable such that the appropriate webpack loader picks it up.

The footer _is_ also missing from my own demo for similar, but not identical reasons! You'll see in my demo at https://surevine.github.io/govuk-react-jsx-examples/ that the logo does work fine with create-react-app out the box. What's going on with the main demo at https://surevine.github.io/govuk-react-jsx/ is I think an unrelated issue with the storybook config which I will sort out separately.

TL;DR - I've fixed your instance of the missing logo, but I will leave this ticket open for a second while I get round to fixing mine! (Probably tomorrow now, it's less pressing than yours)

**Comment by johnbenjaminphe** [comment]
Thank you Andy for your quick response.
